### PR TITLE
Instructions for getting OVS 2.7 with DPDK working on Intel hardware

### PR DESCRIPTION
**Pull request by shivarammysore**

### CODE REVIEW

File: docs/vendors/ovs/README_OVS_2-7_DPDK.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+=======================================================
+
+Detailed OVS connectivity information
+-------------------------------------
+
+.. image:: SDN_Network_Setup.png
+
+**Step 0**:
+
+1.  Install `Ubuntu 17.04 Server <https://www.ubuntu.com/download/server>`_ on a x86 box with OpenSSH enabled and one network management interface enabled.
```
Comment (gizmoguy) [inline docs/vendors/ovs/README_OVS_2-7_DPDK.rst]: OVS-DPDK works on more platforms than this.

File: docs/vendors/ovs/Step1.sh [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+
+
+apt-get install apt-transport-https
+echo "deb https://packages.wand.net.nz $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/wand.list
+echo "deb https://packages.wand.net.nz $(lsb_release -sc)-testing main" | sudo tee /etc/apt/sources.list.d/wand.list
+curl https://packages.wand.net.nz/keyring.gpg -o /etc/apt/trusted.gpg.d/wand.gpg
+apt-get update
+
+apt-get install software-properties-common git wget curl unzip bzip2 screen minicom make gcc dpdk dpdk-dev dpdk-doc dpdk-igb-uio-dkms openvswitch-common openvswitch-switch python-openvswitch openvswitch-pki openvswitch-testcontroller python2.7 libpython2.7 python-pip linux-image-extra-$(uname -r) linux-image-extra-virtual apt-transport-https ca-certificates vlan
```
Comment (gizmoguy) [inline docs/vendors/ovs/Step1.sh]: None of these packages are required.

File: docs/vendors/ovs/Step1.sh [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+apt-get install openvswitch-switch-dpdk
+
+echo "Check installed versions of OpenVSwitch ..."
+dpkg -l openvswitch-common openvswitch-pki openvswitch-switch python-openvswitch openvswitch-switch-dpdk
+
+## Optionally add sensors package for finding out temperature
+apt-get install hwinfo lm-sensors  hddtemp
```
Comment (gizmoguy) [inline docs/vendors/ovs/Step1.sh]: This isn't required for OVS-DPDK.

File: docs/vendors/ovs/Step1.sh [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+service kmod start
+sensors-detect --auto
+sensors
+hddtemp /dev/sda
+
+# enable IPv6
+sysctl net.ipv6.conf.all.disable_ipv6=0
```
Comment (gizmoguy) [inline docs/vendors/ovs/Step1.sh]: IPv6 is enabled by default in Linux

File: docs/vendors/ovs/Step1.sh [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+## fix interface name to suit your machine
+echo  "#iface enp1s0f0 inet6 dhcp" >> /etc/network/interfaces
```
Comment (gizmoguy) [inline docs/vendors/ovs/Step1.sh]: This adds a commented out line?

File: docs/vendors/ovs/Step1.sh [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+## enable interafce config changes by: systemctl restart networking
+## Zero out interface (ex. eth5) attached to the bridge
+# ip addr add 0 dev eth5
+
+echo "Loading the 8021q module into the kernel."
+modprobe 8021q
```
Comment (gizmoguy) [inline docs/vendors/ovs/Step1.sh]: Why is this required for OVS-DPDK?

File: docs/vendors/ovs/Step3.sh [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+# So, assign a hugepage to CPU1
+ovs-vsctl --no-wait set Open_vSwitch . other_config:dpdk-socket-mem="1024,1024"
+ovs-vsctl --no-wait set Open_vSwitch . other_config:dpdk-init=true
+update-alternatives --set ovs-vswitchd /usr/lib/openvswitch-switch-dpdk/ovs-vswitchd-dpdk
+ovs-vsctl add-br ovs-ip64-br0 -- set bridge ovs-ip64-br0 datapath_type=netdev protocols=OpenFlow13 other_config:datapath-id=fa:ce:de:af:ca:fe:ba:be
+ip addr add 10.20.8.8 dev ovs-ip64-br0
+ovs-vsctl show
+## http://docs.openvswitch.org/en/latest/howto/dpdk/
+
+#ovs-vsctl add-port ovs-ip64-br0 dpdk-p0 -- set Interface dpdk-p0 type=dpdk options:dpdk-devargs=0000:82:00.0
```
Comment (gizmoguy) [inline docs/vendors/ovs/Step3.sh]: Why are these commented out lines included in the script?

**Comment by shivarammysore** [comment]
Hi @gizmoguy you can reject this PR.  I removed the same.  I have updated the original document as suggested.  Will submit separate PR for that.
